playbooks/openstack-caasp_instance.yml

```diff
@@ -116,8 +116,8 @@
               {% endfor %}
 
               airship-openstack-control-workers:
-                hosts: &firsttwoworkers
-              {% for node in workers %}{% if loop.index <= 2 %}
+                hosts: &first_workers
```

Comment:
What is actually the meaning of this? I do not understand the syntax...

Reply:
This is a yaml "anchor" 
https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html

Reply:
Thanks!